apps/autonolas-registry/common-util/Details/DetailsSubInfo.jsx

```diff
@@ -121,6 +125,19 @@ export const DetailsSubInfo = ({
     operatorStatusValue,
   } = useOperatorWhitelistComponent(id);
 
+  // resolve ens name
+  useEffect(() => {
```

Comment:
have you considered using just this hook, would that be easier? https://wagmi.sh/react/api/hooks/useEnsName

Reply:
great suggestion, will implement shortly